src/utils/image.js

```diff
@@ -251,3 +136,118 @@ export async function getImages() {
 	images.sort((a, b) => b.date - a.date);
 	return images;
 }
+
+export async function uploadImages(
+	files,
+	Toast,
+	setNoPick,
+	setProgress,
+	setUploading,
+	setImages,
+	next,
+	resolve
+) {
```

Comment:
same here, too many parameters, without types it's not very clear what is where and why and how